LagFreeScreenshots/API/MetadataV2.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using UnityEngine;
+using CVRPlayerEntity = ABI_RC.Core.Player.CVRPlayerEntity;
+using MetaPort = ABI_RC.Core.Savior.MetaPort;
+
+namespace LagFreeScreenshots.API
+{
+    public class MetadataV2
```

Comment:
The V2 api _could_ be renamed theoretically because the old version doesn't exist anymore